arma3/@task_force_radio/addons/task_force_radio/functions/common.sqf

```diff
@@ -21,7 +21,7 @@ TF_CODE_OFFSET = TF_SW_STEREO_OFFSET + 1;
 TF_ADDITIONAL_CHANNEL_OFFSET = TF_CODE_OFFSET + 1;
 TF_ADDITIONAL_STEREO_OFFSET = TF_ADDITIONAL_CHANNEL_OFFSET + 1;
 
-TF_LR_SPEAKER_OFFSET = 7;
+TF_LR_SPEAKER_OFFSET = 8;
```

Comment:
Been a while since I've looked at the code but I'd assume it's because of the on/off option being ahead of it in the array.